packages/daf-core/src/agent.ts

```diff
@@ -132,4 +138,38 @@ export class Agent extends EventEmitter {
     debug('Handle action %o', action)
     return this.actionHandler.handleAction(action, this)
   }
+
```

Comment:
I don't think we need to add all these methods just for GraphQL resolvers. This can be solved by passing `authenticatedDid` in GraphQL context. As explained in my other comment